pkg/component/module.go

```diff
@@ -217,15 +240,38 @@ func (mp *ModuleParameter) IsSameType(other ModuleParameter) bool {
 
 // ResolvedParams resolves paramaters for a module. It returns a JSON encoded
 // string of component parameters.
-func (m *FilesystemModule) ResolvedParams() (string, error) {
+func (m *FilesystemModule) ResolvedParams(envName string) (string, error) {
 	s, err := m.readParams()
 	if err != nil {
 		return "", err
 	}
 
-	object, err := jsonnet.Parse("params.libsonnet", s)
+	envCode, err := params.JsonnetEnvObject(m.app, envName)
 	if err != nil {
-		return "", err
+		return "", errors.Wrap(err, "building environment argument")
+	}
+
+	vm := jsonnet.NewVM()
+	vm.AddJPath(
+		filepath.Join(m.app.Root(), "vendor"),
```

Comment:
Are there any circumstances where we would need the _revendored_ (versioned) packages available here?

Reply:
Is there currently an API for determining this?

Reply:
There's a non-exported function called `revendorPackages` which does revendoring and returns a path to import. But before going down that path, I was asking if the parameter resolution we're trying to do here would rely on installed packages?

Reply:
It knows nothing about packages. It only looks in vendor.

Reply:
Let me rephrase the question - would the jsonnet in `params.jsonnet` depend on code in vendored packages? If not, why do we include `vendor` in the JPath? If yes, it sounds like we should support the revendoring logic so that `params.jsonnet` can refer to unversioned package imports.

Reply:
Regardless of the answer, we can address this separately if needed.